core/src/lib/code-generation/custom-snippet.ts

```diff
@@ -97,6 +97,12 @@ export function initCustomSnippet(): CustomSnippets {
             codeBuilder: QueuedCodeBuilder,
             options: IPageObjectBuilderOptions,
         ) => {
+            const isCamelArrayId = Utils.isCamelArrayId.test(element.id);
+            const hasParameters = element.getterFunction!.parameters.length !== 0;
+            if (hasParameters || isCamelArrayId) {
+                console.log('no Support for arraylike elements in FillForm');
```

Comment:
You really want to log this normally to the console?
I would rather add this to the documentation maybe?
Or is there other stuff you log just like this to the console?